frontend/src/scenes/session-recordings/SessionPlayerDrawer.tsx

```diff
@@ -27,12 +27,12 @@ export function SessionPlayerDrawer({ isPersonPage = false, onClose }: SessionPl
     if (isSessionRecordingsPlayerV3) {
         return (
             <LemonModal isOpen={!!activeSessionRecordingId} onClose={onClose} simple title={''}>
-                <header className="border-b">
+                <header>
                     {activeSessionRecordingId ? (
                         <PlayerMetaV3 playerKey="drawer" sessionRecordingId={activeSessionRecordingId} />
                     ) : null}
                 </header>
-                <LemonModal.Content>
+                <LemonModal.Content embedded>
```

Comment:
tried adding p-0 but because of classname specificity it is overridden.

Reply:
Yeah thats a tricky one. Seems like a fair solution